.github/codecov.yml

```diff
@@ -0,0 +1,4 @@
+coverage:
```

Comment:
Why do such a operation is needed for disabling the coverage test CI?

Reply:
It is because the status checks prevent us from merging the pull request due to the fact that we don't have test cases for UI components right now. Codecov fails the status checks because of the decreasing coverage.

We need to disable the coverage status checks during the early stage of the development. We will add test cases for the user interface later and enable the status checks again.

Note that the test coverage is still important. Please write unit tests for your code.

Reply:
I see, thank you.